packages/camera/camera/pubspec.yaml

```
@@ -8,6 +8,8 @@ homepage: https://github.com/flutter/plugins/tree/master/packages/camera/camera
 dependencies:
   flutter:
     sdk: flutter
+  camera_platform_interface:
+    path: ../camera_platform_interface
```

Add a TODO to make sure we don't forget to change the dependency back to a pub.dev import:

```suggestion
  # TODO: Import from pub.dev once the camera_platform_interface is published. 
  camera_platform_interface:
    path: ../camera_platform_interface
```